src/components/application_manager/test/mobile_message_handler_v1_test.cc

```diff
@@ -75,10 +75,10 @@ TEST(MobileMessageHandlerTestV1Test,
                                                        data_json.length(),
                                                        false);
 
-  application_manager::Message* ptr =
-      MobileMessageHandler::HandleIncomingMessageProtocol(message);
+  std::shared_ptr<application_manager::Message> ptr(
```

Comment:
@SNiukalov Can't see any sharing here. What is the reason to create shared_ptr?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e